src/pages/ScreeningPage.vue

```diff
@@ -1,28 +1,161 @@
 <template>
   <div class="page-wrapper">
-   <ActionBar name="Screenings"/>
-   <ScreeningsHeader sizeType="large"/>
-   <ScreeningsList/>
+    <ActionBar name="Screenings" />
+    <ScreeningsHeader sizeType="large" />
+    <div class="screen-action">
+      <div class="screen-action__days">
+        <DateButton
+          v-for="item in screeningsDays"
+          :caption="item.name"
+          :valueDate="item.date"
+          :key="item.name"
+        />
+
+      </div>
+      <div class="screen-action__select">
+        <label for="select">
+          <span class="select__label">Movie</span>
+          <select
+            name="movies"
+            class="select"
+            v-model="selected"
+            @change="$emit('input', $event.target.value)"
```

Comment:
are you sure that it `ScreeningPage` should emit an event? who is listening to this?